libchisel/src/verifyimports.rs

```diff
@@ -169,7 +169,7 @@ impl<'a> ImportCheck for ImportType<'a> {
                                 ImportStatus::Malformed
                             }
                         } else {
-                            ImportStatus::Malformed
+                            panic!(); // This should never happen.
```

Comment:
Instead of `panic!()` here cannot you just do `let sig = func_sig.unwrap()`?

Reply:
Or even `.expect`

Reply:
Will do